QENSmodels/delta_lorentz.py

```diff
@@ -74,16 +73,33 @@ def sqwDeltaLorentz(
 
     # Input validation
     q = np.asarray(q, dtype=np.float32)
+    A0 = np.asarray(A0)
+    hwhm = np.asarray(hwhm)
 
     # Create output array
     sqw = np.zeros((q.size, w.size))
 
     # Model
     if q.size > 1:
+        # if only a single float is given for A0, adapt to size of q
+        if A0.size == 1:
```

Comment:
`float` and `list` have no attribute `size` and will lead to an error if `A0` is passed as one of those types. maybe include a type check for `A0`

Reply:
`size` and `shape` are used on `numpy.array`. For example, about this comment, `A0` is converted to a `numpy.array` a few lines above (line 76). 
I checked that this was also the case in all models.